classifier.go

```diff
@@ -75,31 +75,50 @@ type License struct {
 	// Threshold is the lowest confidence percentage acceptable for the
 	// classifier.
 	Threshold float64
+
+	// db is the path to the license database
+	db string
+}
+
+// OptionFunc set options on a License struct.
+type OptionFunc func(l *License) error
+
+// Database is an OptionFunc to specify the location of the license database.
+func Database(db string) OptionFunc {
+	// should we check for db existence here?
```

Comment:
Please add a TODO to the start of this comment or remove it.

It might make more sense to do this check in New*() methods though.

Reply:
Removed.  The check happens way deeper in the code.